### PR TITLE
Admin user management

### PR DESCRIPTION
**Pull request by Palusha**

### CODE REVIEW

File: book_storage/app.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+from bson import ObjectId
+from bson.errors import InvalidId
+from dotenv import load_dotenv
+from flask import Flask, render_template, flash, redirect, request, url_for
+from forms import AddUserForm, UpdateUserForm
+from mongoengine import connect
+from mongoengine.errors import NotUniqueError, DoesNotExist, ValidationError
+from werkzeug.security import generate_password_hash
+
+from models import Users
```
Comment (Aarrtteemm123) [inline book_storage/app.py]: Sort by alphabet

File: book_storage/app.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            return redirect(url_for('get_active_users_list'))
+        except (NotUniqueError, ValidationError) as e:
+            print("E-mail already found or user entered invalid e-mail")
+            print(e)
+            return redirect(url_for('create_user'))
+    return render_template('create_user.html', form=form)
+
+
+@app.route('/update_user/<string:_id>', methods=['POST', 'GET'])
+def update_user(_id):
```
Comment (Aarrtteemm123) [inline book_storage/app.py]: add type hinting

File: book_storage/app.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            return redirect(url_for('get_active_users_list'))
+    except (DoesNotExist, InvalidId) as e:
+        print('Such user doesn\'t exist')
+        print(e)
+        return redirect(url_for('get_active_users_list'))
+    return render_template('update_user.html', user=user, form=form)
+
+
+@app.route('/delete_user/<string:_id>')
+def delete_user(_id):
```
Comment (Aarrtteemm123) [inline book_storage/app.py]: add type hinting

File: book_storage/models.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from mongoengine import Document, StringField, EmailField
+
+
+class Users(Document):
+    firstname = StringField(max_length=50, required=True)
+    lastname = StringField(max_length=50, required=True)
+    email = EmailField(required=True, unique=True)
+    password = StringField(required=True)
+    role = StringField(required=True)
+    status = StringField(required=True)
```
Comment (Aarrtteemm123) [inline book_storage/models.py]: Set default value 'active' for field 'status'

File: book_storage/app.py [hunk 5]
```diff
@@ -0,0 +1,102 @@
+import os
+import traceback
+
+from bson import ObjectId
+from dotenv import load_dotenv
+from flask import Flask, render_template, flash, redirect, request, url_for
```
Comment (ivan-dubei) [inline book_storage/app.py]: Sort imports in the `from flask import ` in the alphabetical order

**Review by mehalyna — APPROVED** [review]

**Review by ivan-dubei — APPROVED** [review]

**Review by Aarrtteemm123 — APPROVED** [review]

**Pull request merged**